transmission_interface/src/transmission_loader.cpp

```diff
@@ -53,7 +53,7 @@ TransmissionLoader::getActuatorReduction(const TiXmlElement& parent_el,
       ROS_DEBUG_STREAM_NAMED("parser", "Actuator '" << actuator_name << "' of transmission '" << transmission_name <<
                              "' does not specify the optional <mechanicalReduction> element.");
     }
-    return ParseStatus::NO_DATA;
+    return true;
```

Comment:
I think this should return `false`, at least in the case that `required==true`. That would make this change line up better with the `status != ParseStatus::SUCCESS`->`!status` change.

Reply:
yeah I think I agree w/you there. I'll make the fix

Reply:
Ok I have update it to return false when `required == true`